packages/core/utils/lib/convert-query-params.js

```diff
@@ -218,7 +218,74 @@ const convertFieldsQueryParams = (fields, depth = 0) => {
   throw new Error('Invalid fields parameter. Expected a string or an array of strings');
 };
 
-const convertFiltersQueryParams = filters => filters;
+const convertFiltersQueryParams = (filters, type) => {
+  const sanitizeFilters = (filters, type) => {
+    if (Array.isArray(filters)) {
+      return filters.map(filter => sanitizeFilters(filter, type));
+    }
+
+    for (const operator in filters) {
+      const attribute = type.attributes[operator];
+
+      if (attribute) {
+        if (attribute.type === 'password') {
+          delete filters[operator];
+        }
+
+        if (attribute.type === 'relation') {
```

Comment:
We also have to consider components / medias & dz. Some can be filtered on some should be disabled